src/cpptests/t_agg.cpp

```diff
@@ -15,6 +15,8 @@
 class AggTest : public ::testing::Test {};
 using RS::addDocument;
 
+#if 0
```

Comment:
Why disabling?

Reply:
It's related to `IndexSpec_FreeWithKey()`. If we can replace it with `IndexSpec_Free()` we can put it back.

Reply:
Well, actually it's more due to lack of `RedisModule_ScanCursorCreate()` in redismock.